cli/tests/integration_tests.rs

```diff
@@ -1480,6 +1480,29 @@ itest!(deno_test_only {
   output: "deno_test_only.ts.out",
 });
 
+#[test]
+fn timeout_clear() {
+  // https://github.com/denoland/deno/issues/7599
+
+  use std::time::Duration;
+  use std::time::Instant;
+
+  let mut p = util::deno_cmd()
+    .current_dir(util::tests_path())
+    .arg("run")
+    .arg("--reload")
+    .arg("set_timeout_clear.ts")
+    .spawn()
+    .unwrap();
```

Comment:
Would be cool the read the source code from stdin, so you didn't have to create a separate file `set_timeout_clear.ts`

Reply:
nice work